src/lib/Microsoft.Fx.Portability/Utils/JsonConverters/JsonPublicKeyTokenConverter.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.Fx.Portability.ObjectModel;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Security.Cryptography.X509Certificates;
+using System.Text;
+
+namespace Microsoft.Fx.Portability.Utils.JsonConverters
+{
+    internal class JsonPublicKeyTokenConverter : JsonConverter<PublicKeyToken>
```

Comment:
Where is this serialization used and what is it optimized for? Serializing as a string would be a much more efficient and easy to deal with encoding, but this looks like it would work.

Reply:
It's probably used in returning a response or sending a request. Must not have any tests associated with it so my change wasn't caught.

Reply:
@twsouthwick This PR fix the current problem if both client and server sides using the new binary contains PublicKeyToken struct, but if old version of client without knowledge of PublicKeyToken works with new Server, the serialization will still break. Looks like PR #784 broke backward compat serialization.

Reply:
You should be able to have a converter that would work for both. Do a string serialization (maybe that's what @marklio was referring to) and it should then work for backwards compatibility

Reply:
You can serialize with `PublicKeyToken.ToString()` and deserialize with `PublicKeyToken.Parse(string)`. That should provide round tripping

Reply:
Submitted a new commit. Verified working with backward compat.